code/obj/machinery.dm

```diff
@@ -134,22 +134,25 @@
 
 /obj/machinery/Topic(href, href_list)
 	..()
+	var/mob/living/carbon/human/H = null
+	if(ishuman(usr))
+		H = usr
 	if(status & (NOPOWER|BROKEN))
 		//boutput(usr, "<span class='alert'>That machine is not powered!</span>")
 		return 1
 	if(usr.restrained() || usr.lying || usr.stat)
 		//boutput(usr, "<span class='alert'>You are unable to do that currently!</span>")
 		return 1
 	if(!hasvar(src,"portable") || !src:portable)
-		if ((!in_interact_range(src, usr) || !istype(src.loc, /turf)) && !issilicon(usr) && !isAI(usr))
+		if ((!in_interact_range(src, usr) || !istype(src.loc, /turf)) && !issilicon(usr) && !isAI(usr) && !locate(/obj/item/augmentation/head/wireless_interact) in H?.has_augmentation("Head"))
```

Comment:
this syntax is really weird? locate a typepath in a has_augmentation call return?

Seems really clunky to me, should be wrapped into a call